pkg/proxy/proxy_integration_test.go

```diff
@@ -72,18 +68,6 @@ func TestIntegrationCreateLoadtestFormPostAllFiles(t *testing.T) {
 		err := testhelper.WaitLoadtest(clientSet, createdLoadTestName)
 		require.NoError(t, err)
 	})
-
-	t.Run("Checking if the loadtest testData is correct", func(t *testing.T) {
```

Comment:
Not needed for Fake provider, all cases should be covered in https://github.com/hellofresh/kangal/pull/29